cmd/ceremony/cert.go

```diff
@@ -318,11 +318,17 @@ func makeTemplate(randReader io.Reader, profile *certProfile, pubKey []byte, tbc
 	}
 
 	for _, policyConfig := range profile.Policies {
-		oid, err := parseOID(policyConfig.OID)
+		asnOID, err := parseOID(policyConfig.OID)
 		if err != nil {
 			return nil, err
 		}
-		cert.PolicyIdentifiers = append(cert.PolicyIdentifiers, oid)
+		cert.PolicyIdentifiers = append(cert.PolicyIdentifiers, asnOID)
+
+		x509OID, err := x509.ParseOID(policyConfig.OID)
+		if err != nil {
+			return nil, err
```

Comment:
x509.ParseOID is an external library function, so we should annotate any error it returns with a `fmt.Errorf("some message: %w", err)`.

Reply:
Updated in commit https://github.com/letsencrypt/boulder/pull/7940/commits/4ad9f89b9fd9c3674d124b92610e5da3f3470d52